cloudevents/extensions/jwt.md

```diff
@@ -0,0 +1,147 @@
+# JWT Extension
+
+This extension provides a way to map JWT (JSON Web Token) values in the formats of JWE
+ (JSON Web Encryption) or JWS (JSON Web Signature).
+ 
+JWE MAY be used to encrypt a CloudEvent data and JWS MAY be used to sign CloudEvent data
```

Comment:
Missing `.` at the end of most of the following paragraphs (and this one).

Reply:
I assume "data" here means the `data` property and not "CE metadata", right? If so, let's put back-ticks around `data` to make that clearer.